src/hooks/engine.rs

```diff
@@ -1460,6 +1489,34 @@ pub unsafe fn player_edict(marker: MainThreadMarker) -> Option<NonNull<edict_s>>
     }
 }
 
+pub unsafe fn find_cmd(marker: MainThreadMarker, name: &str) -> Option<NonNull<cmd_function_s>> {
+    let cmd_functions_ = &*cmd_functions.get_opt(marker)?;
+
+    // in case the list is empty
+    if cmd_functions_.is_null() {
+        return None;
+    }
+
+    let mut curr_cmd_function = *cmd_functions_;
+
+    loop {
+        let cmd_function = &*curr_cmd_function;
+        let curr_name = CStr::from_ptr(cmd_function.name as *mut i8);
+
+        if curr_name.to_str().ok()? == name {
```

Comment:
This shouldn't early-return upon to_str() failure, it just means we need to skip this cmd.
```suggestion
        if curr_name.to_str() == Some(name) {
```